src/electricfield.jl

```diff
@@ -1,6 +1,13 @@
 # # Electric Field
 
-function E(r, t, laser)
+E(r, t, laser, symbol) = E(r, t, laser, Val(symbol))
```

Comment:
```suggestion
E(r, t, laser, symbol::Symbol) = E(r, t, laser, Val(symbol))
```
I think this should be restricted here, but I'm not sure if this can lead to method ambiguities.

Reply:
Passed tests with new implementation. Should be good!